cmd/package_managers.go

```diff
@@ -18,12 +18,46 @@ package cmd
 import (
 	"encoding/json"
 	"fmt"
+	"io"
+	"regexp"
 
 	ngt "github.com/ossf/scorecard/v4/cmd/internal/nuget"
 	pmc "github.com/ossf/scorecard/v4/cmd/internal/packagemanager"
 	sce "github.com/ossf/scorecard/v4/errors"
 )
 
+var _GITHUB_DOMAIN_REGEXP = regexp.MustCompile(`^https?://github[.]com/([^/]+)/([^/.]+)`)
+var _GITHUB_SUBDOMAIN_REGEXP = regexp.MustCompile(`^https?://([^.]+)[.]github[.]io/([^/.]+).*`)
+var _GITLAB_DOMAIN_REGEXP = regexp.MustCompile(`^https?://gitlab[.]com/([^/]+)/([^/.]+)`)
```

Comment:
The linter doesn't like multiple things here. It's expecting something more of the form:
```
var (
    githubDomainRegexp = ...
    githubSubdomainRegexp = ...
    gitlabSubdomainRegexp = ...
)
```

Reply:
Ah I just realized i put in CapitalCamelCase instead of camelCase. The linter didn't complain, happy to do whatever you prefer

Reply:
Capitalization causes the symbol to be exported in Go, so would prefer keeping these regexes private.

Reply:
Same comment for the PYPI matcher func array

Reply:
Done!